### PR TITLE
Document How To Control CDN & Browser Cache

### PR DESCRIPTION
**Pull request by nodanaonlyzuul**
Add `--cache-control` flag in documented S3 command.

This dictates cache life on CloudFront edges and users' browsers: 

http://docs.aws.amazon.com/AmazonCloudFront/latest/DeveloperGuide/Expiration.html#ExpirationDownloadDist

### CODE REVIEW

**Comment by EdwinGuzman** [comment]
Looks good. Is 300 an example or do you want to use that for the max-age?

File: README.md [hunk 1]
```diff
@@ -28,7 +28,7 @@ bin/generate_landing_page <spreadsheet_file_name> <output_directory>
 Follow the instructions at http://docs.aws.amazon.com/cli/latest/userguide/installing.html to install and configure the aws cli.
 
 To upload an output directory to an s3 bucket, run
-```aws s3 cp output s3://bucket-name/ --recursive --acl public-read --profile your-aws-cli-profile-name```
+```aws s3 cp output s3://bucket-name/ --recursive --acl --cache-control max-age=300 public-read --profile your-aws-cli-profile-name```
```
Comment (aslagle) [inline README.md]: I think the `--acl public-read` needs to stay together.

Comment (nodanaonlyzuul) [inline README.md]: GREAT CATCH AMY! Yes - that was a silly copy+paste error on my part.
It's been fixed.

**Comment by nodanaonlyzuul** [comment]
@EdwinGuzman:
I put 300 seconds so we can start with short cache while testing and probably for the initial deploy - and can go with longer cache time when we get more confident. Because of the high stakes - I'm scared of permanently caching bad content no 8 million NYer's phones.

**Pull request merged**